README-pl.md

```diff
@@ -0,0 +1,83 @@
+# Dokumentacja Kubernetes
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+Witamy!
+
+W tym repozytorium znajdziesz wszystko, co potrzebujesz do zbudowania [strony internetowej Kubernetes wraz z dokumentacją](https://kubernetes.io/). Bardzo nam miło, że chcesz wziąć w tym udział!
+
+## Twój wkład w dokumentację
+
+Możesz kliknąć w przycisk **Fork** w prawym górnym rogu ekranu, aby stworzyć kopię tego repozytorium na swoim koncie GitHub. Taki rodzaj kopii (odgałęzienia) nazywa się *fork*. Zmieniaj w nim, co chcesz, a kiedy będziesz już gotowy/a przesłać te zmiany do nas, przejdź do swojej kopii i stwórz nowy *pull request*, abyśmy zostali o tym poinformowani.
```

Comment:
Maybe it's worth to create additional page for abbreviations/disambiguation for such phrases as `fork`, `pull request` and so on?

Reply:
We can start with this: https://github.com/nurkiewicz/polski-w-it

Reply:
There is no translation of *fork* there :(

Reply:
Neither *pull request*...

Reply:
If you want to, add some of those terms to https://kubernetes.io/docs/reference/glossary/?community=true

but then _every_ localization has to translate them.

Reply:
I think **Fork** is explained later in the first paragraph. As for pull request, I am not familiar with any good term for that in Polish.

Perhaps we could contribute some vocabulary back to https://github.com/nurkiewicz/polski-w-it ? 

I think translations of newer or less commonly used terms into Polish are a wider problem. From what I've seen closer to me, people just use the English terms where there's no good translation available. And I've seen lack of good translated terms in a lot of places.

I suspect we'll run into quite a few words I always had a hard time translating - such as "packaging".

I'm not sure what's the best option here, but I think this is something we may want to address early on and define how to handle it.